src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,221 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.exception.MissingAtEventException;
+import duke.exception.MissingByDeadlineException;
+import duke.exception.MissingDetailsException;
+import duke.exception.UnknownDateTimeException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.Todo;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Command {
+    //Custom byeCommand Method to exit Duke
+    public static String byeCommand() {
+        return ("BYE");
+    }
+
+    /**
+     * calendarCommand Method finds Tasks that finds, matches and displays Tasks
```

Comment:
In method header comments, the first sentence should start in the form Returns ..., Sends ..., Adds ... 
Suggestion: Finds Tasks that finds, matches and displays Tasks on a specific date.

Reply:
My practice includes having the method's name into the JavaDoc, just as double redundancy when refactoring code (e.g. IntelliJ might automatically rename my actual method name through refactoring), but at least I've disabled it doing the same for JavaDoc or //comments, so I can quickly reference them again in case I "lose a method" (which has happened to me a couple of times), when trying to shift things around.

Regardless, yes, I start immediately with a Verb (e.g. finds, prints, creates) to keep to the convention, especially when searching for a function. Thank you for the suggestion, nonetheless!